resources/views/navbar/hamburger.blade.php

```diff
@@ -23,9 +24,13 @@
         </span>
 
         @isset($allNotificationsIndicator)
-        {{ $allNotificationsIndicator }}
+            {{ $allNotificationsIndicator }}
```

Comment:
does this still work when calling it as `<x-ark-navbar-hamburger :breakpoint="$breakpoint ?? 'md'" />` ?

Reply:
@ItsANameToo mmm that indicator is not currently being used, (Checked all projects), msq has his own indicator component but is being shown in his custom menu, so I'm removing this
![image](https://user-images.githubusercontent.com/17262776/126805290-95a9bec4-dada-4c94-bbd9-f93b2f5775fb.png)